djangoRT/static/djangoRT/css/djangoRT.css

```diff
@@ -59,3 +59,17 @@
 	content: "*";
 	color: red;
 }
+
+details {
+    margin-left: 2em;
+    display: list-item;
```

Comment:
I think the built-in display, which renders a disclosure triangle, communicates better the functionality.

Reply:
Ah yes, I missed this was a `display` attribute. Makes sense. So maybe it's `list-style`.  You could try `list-style-type: revert;` or `list-style-type: unset;` to try to get rid of any overrides being applied elsewhere? Otherwise I think this may work:

```
summary {
  display: list-item;
  list-style: disclosure-closed;
}
details[open] > summary {
  list-style: disclosure-open;
}
```